### ISSUE TITLE
issue with model  last layer 

### ISSUE THREAD

**Issue by speeding-motor** [allanzelener/YAD2K#185]
in keras_darknet19.py

```
def darknet19(inputs):
    """Generate Darknet-19 model for Imagenet classification."""
    body = darknet_body()(inputs)
    logits = DarknetConv2D(1000, (1, 1), activation='softmax')(body)
    return Model(inputs, logits)
```
question :
1、here 1000 = anchor_num * (5 + class_num)  in other dataset?
2、and why use activation softmax here? 

I'm really confused with that, any help is appreciate ,thanks in advance
